### DISCUSSION TITLE
Dialogic 1.4.1 Crashing on Android

### DISCUSSION THREAD

**Discussion by frederickiv43** [dialogic-godot/dialogic#879] [Q&A] [answered]
Hello all,

I've been having some trouble with Dialogic 1.4.1. On my PC it works flawlessly. However, when I export to Android (Samsung Galaxy s8) to debug it will freeze and crash the whole game every time I try to call a dialog. The error is as follows.

[D] An error occurred when trying to access the custom event folder

![Dialogic1 4 1](https://user-images.githubusercontent.com/81455079/166085326-c2f32bd8-93e7-4da6-b06d-e0a9068a5e07.png)

This is even after removing and redownloading the addon, both with and without a custom theme. Any idea what could be going on? Thanks!

**Comment by zaknafean** [comment, marked as answer]
So the 'custom event folder' bug is a bit of a red herring. It's annoying, but not causing the problem. You're not the first to suffer from this as #841 shows, and I recently got hit with it as well. I think I found the solution, and it should be going into 1.4.2 soon. But if you want to see if it works for you immediately 

Find **export_plugin.gd**, which should be at **res://addons/dialogic/Other/export_plugin.gd** and add the following starting at line 4 (first thing after the function declaration)

```
# This fixes the android export 'multiple file in zip' bug
	for feature in features:
		if feature == "Android":
			return
```

In addition make sure you're export configuration contains the following
![image](https://user-images.githubusercontent.com/7741797/166085624-109127f8-37d5-4574-bceb-b167d7354bb4.png)

If this works for you, please let us know!

If it does work you may also be interested in #876 since right now advancing dialog in mobile is broken

**Reply by zaknafean** [reply]
I got hit with the bug I think, and so this fix probably won't help you.

Are you actually using custom events at all?

**Comment by frederickiv43** [comment]
Hey thanks for replying so quick! That fix actually worked for me and I can advance my dialog using a touch button that's called to show/hide when the timeline begins/ends.

I haven't used custom events at all. I'm still getting the 

[D] An error occurred when trying to access the custom event folder

message but at least it's not crashing me anymore.
